.github/workflows/saml.yml

```diff
@@ -1,7 +1,25 @@
 # Workflow starts automatically on push or can be triggered manually.
 # This is a desirable pattern as it allows for adhoc test runs without a code push
 name: saml test
-on: [workflow_dispatch, repository_dispatch]
+on:
+  workflow_dispatch:
+    inputs:
+      filename:
```

Comment:
I think these filename variables can be removed

Reply:
yes, copy and paste error :)